src/transformers/modeling_roberta.py

```diff
@@ -175,6 +181,123 @@ def set_input_embeddings(self, value):
         self.embeddings.word_embeddings = value
 
 
+@add_start_docstrings(
+    """RoBERTa Model with a `language modeling` head on top for CLM fine-tuning. """, ROBERTA_START_DOCSTRING
+)
+class RobertaForCausalLM(BertPreTrainedModel):
```

Comment:
Wouldn't it be more coherent to have it as `RobertaLMHeadModel`?

Reply:
(But I do prefer `RobertaForCausalLM`)

Reply:
It's the same terminology for BERT.

Reply:
Following internal discussion will leave the name as it is more precise and `BertLMHeadModel` should change in the future.